### ISSUE TITLE
installation downgrades packages

### ISSUE THREAD

**Issue by cardosan** [LCA-ActivityBrowser/activity-browser#31]
Hi guys,
first happy to see you take the development back.

I have just installed `AB` in an already created environment and some packages got downgraded

```sh
The following NEW packages will be INSTALLED:

    activity-browser: 2.2.1-pyhd50db26_1 haasad     

The following packages will be UPDATED:

    pyqt:             5.6.0-py36_4       conda-forge --> 5.9.2-py36h7ca782d_0  haasad
    sip:              4.18-py36_1        conda-forge --> 4.19.6-py36h7b295a5_1 haasad

The following packages will be SUPERSEDED by a higher-priority channel:

    fontconfig:       2.12.1-4           conda-forge --> 2.11.1-6                    
    freetype:         2.7-1              conda-forge --> 2.5.5-2                     
    matplotlib:       2.1.0-py36_0       conda-forge --> 2.0.2-np113py36_0           
    qt:               5.6.2-3            conda-forge --> 4.8.7-3                     

```

Dunno if this unwanted behaviour or  is necessary to make `AB` works, but in the latter case I would specify in the section`Install the activity browser` of the `README` that `AB` must/should be installed in a separate environment to avoid this.

**Comment by haasad** [comment]
Hi Giuseppe,
this is related to #16. Downgrading should have no visible effect for you.

I can try to explain what happens here:
- With the previous installation pyqt required qt5, but since my package includes qt with pyqt, qt 5 is now no longer explicitly required
- Because you don't have an explicit requirement for qt5 anymore, conda looks for qt in your highest priority channel (`defaults`), where it finds qt 4.8
- This triggers a chain reaction where some other packages from `conda-forge` are downgraded to the newest version in `defaults` (this is a somewhat new behaviour of conda describe [here](https://conda.io/docs/user-guide/tasks/manage-channels.html).

TLDR: Expected behaviour

I agree that some more info in the README would be useful here. Will add something before closing this issue.

**Comment by haasad** [comment]
Updated the README. Thanks for raising the issue. Really cool that you already start testing the new installation just a few hours after I added the instructions!

**Comment by cardosan** [comment]
I just started a new work where I will have to modify several dbs and this release came at the right time...it is so much faster to use AB to browse around dbs....it means that other feedbacks might come :)
